### PR TITLE
DOCSP-34684 v2.1.1 release notes

### PR DESCRIPTION
**Pull request by jmd-mongo**
## DESCRIPTION

v2.1.1 release notes.

## STAGING

https://preview-mongodbjmdmongo.gatsbyjs.io/mongodb-shell/DOCSP-34684/changelog/#v2.1.1

## JIRA

https://jira.mongodb.org/browse/DOCSP-34684

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66339542b67dbf804a4752e1

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Is this free of spelling errors?
- [X] Is this free of grammatical errors?
- [X] Is this free of staging / rendering issues?
- [X] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -192,6 +192,23 @@ v2.1.3
...
+  version conflict on RHEL8 distros 
+- :issue:`MONGOSH-1651` - Sample sessions sent to telemetry
+- :issue:`MONGOSH-1652` - Disable initial showCommands from telemetry
+- :issue:`MONGOSH-1653` - Add a new "Startup Time" event for sessions
+- :issue:`MONGOSH-1654` - Re-enable arm64 executable connectivity tests
+- :issue:`MONGOSH-1656` - Allow startup timings to be grouped
+- :issue:`MONGOSH-1658` - node-worker-runtime: remove function 
+  properties from errors before serialization
+
+`Full release notes available on JIRA <https://jira.mongodb.org/secure/ConfigureReleaseNote.jspa?projectId=17381&version=37380>`__.
```
Comment (ltran-mdb2) [inline source/changelog.txt:210]: [question] Should this link lead to the JIRA tickets as opposed to the page to configure the release notes?

**Review by ltran-mdb2 — APPROVED** [review]
LGTM mod one question! Thanks @jmd-mongo !

**Comment by jmd-mongo** [comment]
Hi Anna -- sorry for the substantial delay on this on. Could you please review when you have a chance?

Thank you!
Joe

File: source/changelog.txt [hunk 2]
```diff
@@ -192,6 +192,23 @@ v2.1.3
...
+
+- :issue:`MONGOSH-1628` - Statically linking kerberos leads to OpenSSL 
+  version conflict on RHEL8 distros 
+- :issue:`MONGOSH-1651` - Sample sessions sent to telemetry
+- :issue:`MONGOSH-1652` - Disable initial showCommands from telemetry
+- :issue:`MONGOSH-1653` - Add a new "Startup Time" event for sessions
+- :issue:`MONGOSH-1654` - Re-enable arm64 executable connectivity tests
+- :issue:`MONGOSH-1656` - Allow startup timings to be grouped
+- :issue:`MONGOSH-1658` - node-worker-runtime: remove function 
+  properties from errors before serialization
```
Comment (addaleax) [inline source/changelog.txt:208]:
```suggestion
```

I'd drop these, none of them have user-facing impact

**Review by addaleax — APPROVED** [review]

**Pull request merged**